client/my-sites/plans-single-products/product-price-group.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import PlanPrice from 'my-sites/plan-price';
+
+const PlansSingleProductPriceGroup = props => {
+	const { billingTimeFrame, currencyCode, discountedPrice, fullPrice, isPlaceholder } = props;
+	const isDiscounted = !! discountedPrice;
+	const priceGroupClasses = classNames( 'plans-single-products__price-group', {
+		'is-discounted': isDiscounted,
+		'is-placeholder': isPlaceholder,
+	} );
+
+	if ( ! isDiscounted ) {
```

Comment:
Seems like this block almost duplicates the following one; does it make sense to display the discounted price only if we have a discounted product?

Reply:
We could, but we should then also add the `original` and `discounted` props conditionally. I thought this approach might be less readable than having just two return statements, but I'm happy to try that other approach out in the next PR 👍